### ISSUE TITLE
Inventory all issues related to indexable structures

### ISSUE THREAD

**Issue by JacquesCarette** [JacquesCarette/Drasil#3717]
All open issues that mention indexing, vectors, matrices, should be found and linked. This is part of the requirements analysis phase of this project.

**Comment by J1aM1ng** [comment]
@JacquesCarette The relevant open issues I was able to find are as follows:

#2587 (indexable vectors, valuable to do),
#359 (index for symbols, handling of some edge cases),
#1226 (indexed vectors, previously pending due to insufficient prioritization),
#1558 (extraction of vector information),
#1801 (we may need to evaluate whether it is necessary to deal with the case of switching between two constraints),
#1814 (definition? about displacement unit vector),
#3135 (about the size of vectors and metrices?),
#3148 (about vector related case study),
#3149 (about vector operation),
#3246 (reconstruct the Real space of physical quantities in drasil-data?)
#3508 (about indexing of arrays in GOOL? I'm not sure if we should put it here),
#3572 (add more knowledge on vectors to double pendulum?),
#1539 (label in Traceability Matrix) and,
#3567 (order in the traceability matrix may be used as a solution for the order of labels in the RefBy).

If there are any omissions, please feel free to add them.

These related open issues seem very diverse, should we take the time to discuss them at Friday's Drasil meeting?

**Comment by JacquesCarette** [comment]
You missed #3746 and #3747 -- aren't you looking at the recent discussions as well? There might be things on the wiki too. Those issues certainly form a good starting point.

**Comment by J1aM1ng** [comment]
Yes, #3746 and #3747 should also be added here, as these two issues recently discussed the implementation of indexing of arrays.

**Comment by balacij** [comment]
@J1aM1ng There's also tickets from the [Improving Vector Support](https://github.com/JacquesCarette/Drasil/projects/17) project. In fact, I think the new project is a duplicate of that one. Can we reconcile them (let's go towards the new one however, since Classic projects are being removed soon)? Also, I think you can link those tickets to the actual project via the RHS "Projects" section, similar to how @JacquesCarette linked this ticket to the new "Vectors, Matrices, Arays, etc." project. After linking them, I suppose we can close this issue?

Note: I don't have permissions to edit the new project, so I can't do this. However, one suggestion I have is to create a "Symptoms" / "Research" / "Blocks" column (depending on how you want to view if the tickets suggest we need to improve vector support, or if you view the tickets as something we can unblock by improving vector support). Many of those tickets will be work tasks related to your project eventually, I think.
